detox/android/detox/src/main/java/com/wix/detox/espresso/action/DetoxCustomTapper.kt

```diff
@@ -104,8 +106,8 @@ open class DetoxMultiTap
 
     private fun verifyTapEventTimes(upEvent: CallInfo, downEvent: CallInfo) {
         val delta: Long = (upEvent - downEvent)!!
-        if (delta >= longTapMinTimeMs) {
+        if (delta >= longTapMinTimeMs && duration == null) {
```

Comment:
Maybe split into 2 classes, one dedicated to long press related things (w/ or w/o duration).